astropy/cosmology/io/_parameter.py

```diff
@@ -0,0 +1,74 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+
+from astropy.cosmology.parameter import Parameter
+from astropy.table import Column
+from astropy.modeling import Parameter as ModelParameter
+
+
+def _convert_Parameter_to_Column(parameter, value, param_meta=None):
+    """Convert a Cosmology Parameter to a Table Column.
+
+    Parameters
+    ----------
+    parameter : `astropy.cosmology.parameter.Parameter`
+    value : Any
+    meta : dict or None, optional
+        Information from the Cosmology's metadata.
+
+    Returns
+    -------
+    `astropy.table.Column`
+    """
+    if value is None:  # Create None column
+        data = [None]
+        dtype = np.object_
+        format = None
+    else:  # Create empty column. Assign later.
+        data = None  # assigned later
```

Comment:
Why does it not work to just pass on the `value`? Is it that the shape can be wrong? Maybe do `data = np.reshape(data, (1,)+np.shape(data))`? With `data=dtype=None`, you get `float`. I assume that is always fine?

Reply:
The above is actually to avoid dtype float. ECSV saves that as Nan, so it doesn't round trip correctly.

Reply:
I'm confused `Column(data=None, dtype=None, length=1)` definitely gives a float dtype. (I didn't mean the object dtype in the first part)

Reply:
Ah yes. The shape can be wrong, which is why I was deferring the assignment.
But you're right. The current implementation is getting the dtype wrong.
Thanks for the catch.

Reply:
``np.reshape(data, (1,)+np.shape(data))`` works great, even for structured data!